lua/conform/formatters/rustfmt.lua

```diff
@@ -10,9 +10,11 @@ return {
   options = {
     -- The default edition of Rust to use when no Cargo.toml file is found
     default_edition = "2021",
+    default_toolchain = "stable",
```

Comment:
The `default_edition` is only a default because it's what we use if there is no Cargo.toml file. Let's rename this to just `toolchain`.